client/luigi-client.d.ts

```diff
@@ -612,10 +612,19 @@ export type getEventData = () => Context;
  */
 export function getContext(): Context;
 export type getContext = () => Context;
-
+/**
+ * Set search query parameters to Luigi core. It will be added to url.
+ * @param {Object} searchParams
```

Comment:
**function addNodeParams(searchParam**, ...
Would be good to set **searchParam** to searchParams, just to be consistent with other declarations.